tox.ini

```diff
@@ -21,7 +21,7 @@ commands =
 extras = dev
 commands =
     coverage erase
-    coverage run -m pytest tests/integration {posargs}
+    coverage run -m pytest -s tests/integration {posargs}
```

Comment:
Maybe revert?

Reply:
I've removed all instances of tests printing to stdout so it doesn't really matter but sometimes those sneak in and clobber the CI test output of passing tests (stdout is still printed by pytest if a test fails)